package.json

```diff
@@ -129,7 +129,7 @@
     "postcss-preset-env": "^8.3.0",
     "prettier": "^2.4.1",
     "prop-types": "^15.7.2",
-    "react": "18.2.0",
+    "react": "^18.2.0",
```

Comment:
this one might be fine? it's in dev dependencies whereas the others are in peer dependencies which is what I think the problem is. pretty sure you're allowed to be as specific as you want with your own dev dependencies.

Reply:
undoing it because I don't have a good reason to change it

Reply:
on 2nd/3rd thought..... looking at all the historic values, we've always opted to use `^` as a prefix here

my same logic of sticking to what we were doing before without a good reason to change it might instead suggest that we do go with `^18.2.0` here. I could be convinced otherwise but as far as I know we don't have a need for this to be so specific.